src/features/ajax/instrument/index.js

```diff
@@ -208,17 +208,27 @@ function subscribeToEvents (agentIdentifier, ee, handler, dt) {
   function onFetchBeforeStart (args) {
     var opts = args[1] || {}
     var url
-    // argument is USVString
     if (typeof args[0] === 'string') {
+      // argument is USVString
       url = args[0]
-      // argument is Request object
+
+      if (url.length === 0 && isBrowserScope) {
+        url = '' + globalScope.location.href
+      }
     } else if (args[0] && args[0].url) {
+      // argument is Request object
       url = args[0].url
-      // argument is URL object
     } else if (globalScope?.URL && args[0] && args[0] instanceof URL) {
+      // argument is URL object
       url = args[0].href
     }
 
+    if (typeof url !== 'string' || url.length === 0) {
```

Comment:
Is there any chance for `url` to === `undefined` here and throw an error at `url.length`?